rpc/Cargo.toml

```diff
@@ -81,6 +81,9 @@ version = "17"
 [dependencies.jsonrpc-derive]
 version = "17"
 
+[dependencies.metrics]
+version = "0.15"
```

Comment:
latest is `16.0`?

Reply:
yep, but we'll need to perform a workspace-wide update regardless, and it's better to do that in its own PR